components/CommonHeader.tsx

```diff
@@ -38,7 +52,10 @@ const CommonHeader = () => {
   }
 
   const router = useRouter()
-  const currentRoute = router.route
+
+  const isCurrent = (name: string) => {
+    return name === PATH_TO_SECTION[router.route];
```

Comment:
This is great. To me it makes the logic more explicit and easier for me to follow.